### PR TITLE
Update Release notes for GWT2.8 RC3 and final release

### PR DESCRIPTION
**Pull request by jDramaix**
This change is [Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/203)

### CODE REVIEW

File: src/main/markdown/release-notes.md [hunk 1]
```diff
@@ -59,6 +60,33 @@ The GWT Release Notes
...
+### JsInterop
+- Cleanup/update javadoc for jsinterop annotations.
+- Allow native classes implement native interfaces with overlays.
+
+## <a id="Release_Notes_2_8_0_RC3"></a> Release Notes for 2.8.0 (RC3)
+
+### Bug fixes
+- Fix erroneous constant propagation in JsProperty field.
+- Fix missing bridges for default methods and functionnal expressions.
+- Introduces special `<window>` namespace in JsInterop annotations in order to be able to reference the iframe top scope.
```
Comment (tbroyer) [inline src/main/markdown/release-notes.md]: I thought we didn't want it documented? @rluble @gkdn

File: src/main/markdown/release-notes.md [hunk 2]
```diff
@@ -59,6 +60,33 @@ The GWT Release Notes
...
+- Fix ArrayDeque's bug when expanding capacity.
+- Fix missing declaratio/initialization of JsOverlay fields.
```
Comment (tbroyer) [inline src/main/markdown/release-notes.md]: Typo (missing n)

File: src/main/markdown/release-notes.md [hunk 3]
```diff
@@ -59,6 +60,33 @@ The GWT Release Notes
...
+- Fix generation of methods to access GSS constant in InterfaceGenerator.
+- Fix incorrect property copying for native JsType subclasses.
+- Fix toString dispatch for subtypes of native JsTypes.
+
+### Miscellanous
+- Improve JsInterop error messages on native type fieds.
+- Update globals for Chrome 54.0.2840.16.
+
+### Deprecation
+- Deprecate RootPanel#clear.
```
Comment (tbroyer) [inline src/main/markdown/release-notes.md]: Actually it's only clear(boolean) that's deprecated

File: src/main/markdown/release-notes.md [hunk 4]
```diff
@@ -59,6 +60,33 @@ The GWT Release Notes
...
+## <a id="Release_Notes_2_8_0_RC3"></a> Release Notes for 2.8.0 (RC3)
+
+### Bug fixes
+- Fix erroneous constant propagation in JsProperty field.
+- Fix missing bridges for default methods and functionnal expressions.
+- Introduces special `<window>` namespace in JsInterop annotations in order to be able to reference the iframe top scope.
+- Fix ArrayDeque's bug when expanding capacity.
+- Fix missing declaratio/initialization of JsOverlay fields.
+- Fix missing error reporting in SDM that leads to NPE. 
+- Fix inconsistent naming pattern of method reference classes.
```
Comment (tbroyer) [inline src/main/markdown/release-notes.md]: As a user, I don't quite know what that means

**Comment by gkdn** [comment]
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

*[src/main/markdown/release-notes.md, line 74 at r1](https://reviewable.io:443/reviews/gwtproject/gwt-site/203#-KUUJA0SYmcYWx8390jO-r1-74:-KUXnCsC5ugXdtfYYY-h:b3uhoow) ([raw file](https://github.com/gwtproject/gwt-site/blob/61fb88812942b78de6b63f426d713f9b1219a126/src/main/markdown/release-notes.md#L74)):*
<details><summary><i>Previously, tbroyer (Thomas Broyer) wrote…</i></summary>
> I thought we didn't want it documented? @rluble @gkdn
</details>
pls yes drop this one.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/gwtproject/gwt-site/203)*

File: src/main/markdown/release-notes.md (hunk 3 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: Done

File: src/main/markdown/release-notes.md (hunk 1 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: removed

File: src/main/markdown/release-notes.md (hunk 2 above)
Comment (jDramaix) [inline src/main/markdown/release-notes.md]: done

**Pull request merged**